elixir/apps/web/lib/web/live/policies/index.ex

```diff
@@ -13,7 +13,10 @@ defmodule Web.Policies.Index do
       |> assign_live_table("policies",
         query_module: Policies.Policy.Query,
         sortable_fields: [],
-        hide_filters: [:resource_id, :actor_group_id],
+        hide_filters: [
```

Comment:
@bmanifold can you please find all `assign_live_table("policies"` in the codebase and make sure the filter is disabled where it makes sense?

Reply:
Sure thing.  Didn't realize we might have this anywhere other than the index page for policies.

Reply:
Actually, this is now making me realize that we may need those filters I just deleted.  I see now we have a search input for policies on both Groups and Resources.  If we were on a Resource show page for instance, we'd only want to search Policies by Actor Group name in that instance, and vice versa for Actor Group show page right?

Reply:
Yeah I think that makes sense @bmanifold. Hopefully that unblocks this PR.

Reply:
👍 sounds good.  I'll get the changes made and pushed up

Reply:
Updated